SU-HSDO/suhumsci
# updated to latest version of decanter and updated deprecated code

# READY FOR REVIEW

# Summary
- Update to latest version of Decanter
- Update related deprecated code

# Needed By (Date)
- N/A

# Urgency
- N/A

# Steps to Test

1. Pull this branch and ensure that the latest stanford_basic is added
2. Verify that the latest version of Decanter has been added and that code has been updated (primarily grid-related changes)

# Affected Projects or Products
- suhumsci (theme)

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
